tests/acceptance/live-test.js

```diff
@@ -1,14 +1,36 @@
 import { module, test } from 'qunit';
 import { visit, currentURL } from '@ember/test-helpers';
 import { setupApplicationTest } from 'website-www/tests/helpers';
+import { later } from '@ember/runloop';
 
 module('Acceptance | live', function (hooks) {
   setupApplicationTest(hooks);
 
-  test('visiting /live', async function (assert) {
+  test('skeleton exists!', async function (assert) {
+    assert.expect(3);
     await visit('/live');
 
-    assert.strictEqual(currentURL(), '/live');
+    assert.strictEqual(currentURL(), '/live', 'We are on the live page!');
+
+    assert.dom('[data-test-skeleton]').exists();
+    assert.dom('[data-test-video]').doesNotExist();
+  });
+
+  test('video exists!', async function (assert) {
+    assert.expect(5);
+    await visit('/live');
+
+    const controller = this.owner.lookup('controller:live');
+
+    assert.strictEqual(currentURL(), '/live', 'We are on the live page!');
     assert.dom('[data-test-tabs]').exists();
+
+    await new Promise((resolve) => later(resolve, 4000));
```

Comment:
Don't we have some fakeTimers here? it's not recommended to make the test wait

Reply:
okay

Reply:
Hey @bhtibrewal 
I have changed the approach for tests by using waitFor helper in tests